### ISSUE TITLE
Accordian initial aria-expanded="false' on first item doesn't work as expected

### ISSUE THREAD

**Issue by iamgmd** [themesberg/flowbite#615]
When setting aria-expanded="false" on all item buttons it still opens the first item and causes a double-click needed to expand.

Create any accordion and set all buttons with area-expanded="false"

I expect all items to be collapsed initially.

 - OS: macOS
 - Browser: chrome, safari, brave, etc. 
 - Version: latest

Using Nuxt3
